distrobox-enter

```diff
@@ -364,24 +377,55 @@ if [ "${dryrun}" -ne 0 ]; then
 	exit 0
 fi
 
-# Inspect the container we're working with.
+# Check if the container is even there
+container_exists=$(${container_manager} ps -aq -f name=^"${container_name}"$)
```

Comment:
Could we use the previous `$container_exists` ?
This would let us do all the checks with just 1 `podman/docker inspect` instead of 2, to optimize
`enter` performance

Reply:
If we move all this block after the `eval ...` where `container_exists` was declared before, it should need no other changes